Library/Formula/netdiscover.rb

```diff
@@ -0,0 +1,185 @@
+class Netdiscover < Formula
+  desc "ARP Scanner"
+  homepage "http://sourceforge.net/projects/netdiscover/"
+  url "svn://svn.code.sf.net/p/netdiscover/code/trunk"
+  version "0.3-pre-beta7"
```

Comment:
I'm afraid it'll have to go to our [devel-only tap](https://github.com/Homebrew/homebrew-devel-only) unless there's a stable version.